src/main/java/me/magnet/consultant/Consultant.java

```diff
@@ -559,15 +560,19 @@ public void deregisterService() {
 	}
 
 	public List<ServiceInstance> list(String serviceName) {
-		String url = consulUri + "/v1/health/service/" + serviceName + "?passing&near=_agent";
+		String url = consulUri + "/v1/health/service/" + serviceName + "?passing";
 
 		HttpGet request = new HttpGet(url);
 		request.setHeader("User-Agent", "Consultant");
 		try (CloseableHttpResponse response = http.execute(request)) {
 			int statusCode = response.getStatusLine().getStatusCode();
 			if (statusCode >= 200 && statusCode < 400) {
 				InputStream content = response.getEntity().getContent();
-				return mapper.readValue(content, new TypeReference<List<ServiceInstance>>() {});
+				List<ServiceInstance> instanceLocations =
+						mapper.readValue(content, new TypeReference<List<ServiceInstance>>() {
+						});
+				Collections.shuffle(instanceLocations);
+				return instanceLocations;
```

Comment:
This is just so we are not dependent on the ordering of Consul, should that change in the future